### PR TITLE
Calorie calculator

### LINKED ISSUE #1: Create a drop down list and add form fields for the user to log their exercises.
- the dropdown list will contain a predefined list of exercises to choose from. 
- the form fields is based off of what the api/formula needs 
- there should be a submit button to submit the data
- there should be a database to save this data
- database should also include the information from the drop-down list

### LINKED ISSUE #5: Calculation for calories burned
Maybe use an API to get the calories burned or a formula. 

Get information from what the user submitted to calculate the calories burned. 

The form fields can be changed to whatever you need them to be based on the API/ formula you find most useful.

### PR DESCRIPTION
**Pull request by Arshad-Zaman**
I implemented the calorie calculator and database object for it. Let me know if anything needs to be changed.

### CODE REVIEW

**Review by rcheng2 — COMMENTED** [review]
-

**Review by rcheng2 — APPROVED** [review]
-

File: README.md [hunk 1]
```diff
@@ -1 +0,0 @@
-# FocusFitness
```
Comment (rcheng2) [inline README.md:1]: We have to work on the README as a group

File: app.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+load_dotenv(find_dotenv())
+
+app = flask.Flask(__name__)
+app.config["SECRET_KEY"] = os.getenv("SECRET_KEY")
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+db = SQLAlchemy(app)
+
+class Record(db.Model):
```
Comment (rcheng2) [inline app.py:16]: The database has the fields that we want.

File: templates/index.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```
Comment (rcheng2) [inline templates/index.html:1]: Looks like the figma! Good job

**Comment by eemmanuel2** [comment]
Nice calculator function. Works as intended

**Pull request merged**